mlbench_core/cli/cli.py

```diff
@@ -144,19 +155,24 @@ def run(name, num_workers, gpu, light, dashboard_url):
     text_prompt += "\n\nSelection"
 
     selection = click.prompt(
-        text_prompt, type=click.IntRange(0, len(MLBENCH_BACKENDS)), default=0
+        text_prompt, type=click.IntRange(0, len(MLBENCH_BACKENDS)), default=last_run_inputs.get('backend', 0)
     )
+    current_run_inputs['backend'] = selection
 
     if selection == len(MLBENCH_BACKENDS):
-        backend = click.prompt("Backend", type=str)
+        backend = click.prompt("Backend", type=str, default=last_run_inputs.get('custom_backend', None))
+        current_run_inputs['custom_backend'] = backend
         run_on_all = click.confirm(
-            "Run command on all nodes (otherwise just first node)"
+            "Run command on all nodes (otherwise just first node)", default=last_run_inputs.get('run_on_all', None)
```

Comment:
Not sure if the default value should be `None`. Have you tried it ?

Reply:
yes, I tested it and passing `None` results in no default value, so the user has to enter the value, just like it is now.